rocky/components/modal/script.js

```diff
@@ -0,0 +1,30 @@
+import { onDomReady } from "../js/imports/utils.js";
+
+onDomReady(initDialogs);
+
+export function initDialogs() {
+  let modal_components = document.querySelectorAll(".modal-wrapper");
```

Comment:
We can easily allow for an optional root element to be selected, this is useful when a dev later on adds more dialogs to a page dynamically we can then allow them to more specifcally init those dialogs by calling initDialogs again with their parent.
If we where to also make a function to init a single dialog instead of having that as a anonymous function in the loop on line 8 we can offer them even better performance.
```suggestion
export function initDialogs(element) {
  let root = element || document;
  let modal_components = root.querySelectorAll(".modal-wrapper");
```